community/moderator_sop.md

```diff
@@ -0,0 +1,134 @@
+# Moderation Standard Operating Procedures
+
+The main objective of this SOP is to align our moderation procedures with Operation Code’s purpose:
+
+"A non-profit charity helping the military community learn software development, enter the tech industry, and code the future."
+
+Good guidelines protect our members and community culture. Great guidelines will push our community forward. This is a living document and PR’s for improvements are encouraged, if you are interested in becoming a moderator please sign up as a volunteer at https://op.co.de/nVvNJw
```

Comment:
This https://op.co.de/nVvNJw link is a placeholder for the real one (not sure why it's a 404 right now)

Reply:
https://op.co.de/volunteer-signup should work!